### PR TITLE
actually start deleting old s3 objects

### PR DESCRIPTION
**Pull request by terrazoon**
## Description

Previously we added this ability in "log only" mode.  On staging we can now see there are lots of old s3 objects in need of deletion and the logging seems to work properly.

Modify the code so it actually starts deleting old objects and enhance the test so it proves only old objects, and not new objects, are deleted.

## Security Considerations

N/A

### CODE REVIEW

File: app/aws/s3.py [hunk 1]
```diff
@@ -96,13 +95,18 @@ def cleanup_old_s3_objects():
...
             for obj in response.get("Contents", []):
                 if obj["LastModified"] <= time_limit:
-                    current_app.logger.info(
-                        f"#delete-old-s3-objects Wanting to delete: {obj['LastModified']} {obj['Key']}"
-                    )
+
+                    try:
+                        remove_csv_object(obj["Key"])
+                        current_app.logger.info(
+                            f"#delete-old-s3-objects Deleted: {obj['LastModified']} {obj['Key']}"
```
Comment (ccostino) [inline app/aws/s3.py:105]: Thanks for still logging what's deleted here when it's successful, this way we have a record of the action and it'll still be possible to troubleshoot potential issues that come up with any missing files that we think _should_ be there!

**Review by ccostino — APPROVED** [review]
Thank you, @terrazoon, and especially for the log statements of successful or failed activity and the notes in the test to prove out the timing of the removal!

File: app/config.py [hunk 2]
```diff
@@ -251,7 +251,7 @@ class Config(object):
             },
             "delete_old_s3_objects": {
                 "task": "delete-old-s3-objects",
-                "schedule": crontab(minute="*/5"),
+                "schedule": crontab(hour=7, minute=10),
```
Comment (A-Shumway42) [inline app/config.py:254]: This runs the task less often correct? Just wanted to understand what this change was for.

**Review by A-Shumway42 — APPROVED** [review]
Just had one question for insight but otherwise LGTM!

File: app/config.py (hunk 2 above)
Comment (terrazoon) [reply app/config.py:254]: Yes, run the task once a day instead of every 5 minutes.  Every 5 minutes was just for debugging.

**Pull request merged**